### ISSUE TITLE
Implement "no aerosol" option in scream v1

### ISSUE THREAD

**Issue by hassanbeydoun** [E3SM-Project/scream#1707]
v0 had an option of turning off aerosols by setting do_aerosol_rad and micro_aerosolactivation to false. The former bypasses aerosol calculations in radiation.F90 and sets aerosol optical properties to zeros while the latter sets nc to a constant value saved in p3_utils. Having this option in v1 will allow comparisons with v0 without SPA.  

The v1 p3 code still uses do_prescribed_CCN and do_predict_nc so we can set those to false and have our constant drop number option going. The radiation however does not seem to have a "do_aerosol_rad" equivalent so more work might be needed there. 

@AaronDonahue @brhillman @PeterCaldwell let me know if you have any thoughts. I'll post updates as I have them.

**Comment by PeterCaldwell** [comment]
Additional context - I see this innocuous issue as the path forward for getting ne512 and ne1024 working in the short term (by bypassing the memory bottleneck in SPA init). So I think it is high priority. In my mind, this should be easy but requires changes to our user interface that I don't know how to do - basically, we need to create an option that bundles a couple of user interface options: do_prescribed_CCN=False, do_predict_nc=False, and aerosol optical properties should all be initialized to zero. We also need to specify the prescribed nc value we want to use... I'm not sure if that is possible yet. 

Do you have comments on how this could be done, @jgfouca , @bartgol , @AaronDonahue ?

**Comment by AaronDonahue** [comment]
I've been chatting with @hassanbeydoun on Slack about this...  Some thoughts so far:
1. Adding `do_prescribed_CCN` and `do_predict_nc` as runtime options should be fairly straightforward.  They can just be added to the parameter list in the `namelist_defaults.xml` file for SCREAM.
2. We will need to put some checks into rrtmgp to essentially grab 0 valued arrays when aerosol is turned off.  Note, P3 already has some infrastructure for handling cases when aero is turned off so we just need to pass the correct logical there.
3. I think, that in cases where all of the `aero_blah_sw/lw` variables are going to be unused, we shouldn't even add them to the field manager.  Probably the same with `nccn`.  I think we will just need to add conditionals in the p3 and rrtmgp interfaces to either get those fields when active, or later on assign those fields (as they are used) to some zero array.

**Comment by jgfouca** [comment]
@PeterCaldwell , you mentioned on the telecon that you wanted me to help with this. I can definitely help with point 1 from @AaronDonahue 's list. I can potentially help with 2,3 too but I would need just a bit of guidance as far as what files to look at.

**Comment by AaronDonahue** [comment]
@jgfouca , I've started a branch which addresses (1) above, I'm testing now.  We can go from there.  We will need to do something similar for the rrtmgp variables.  For the moment I'm just focusing on p3.

**Comment by bartgol** [comment]
Actually, using global_parameter entries as selectors complicates things (you'd have to keep both lines in the XML file, in case the user atmchanges the global parameter). But using them inside the parameter value is prob ok. Perhaps if we could have a "ternary" operator inside the XML entry, like
```
  <atm_procs_list>(shoc,cld,$(@use_spa@ ? spa, : ), p3)</atm_procs_list>
```
which evaluates to either `(shoc,cld,spa,p3)` or `(shoc,cld,p3)`, depending on the value of `use_spa` _at yaml generation time_.

**Comment by AaronDonahue** [comment]
@bartgol , I like this suggestion.  For example in the case of this issue.  If we turn off SPA it would be good to automatically set `do_prescribed_ccn` to FALSE, otherwise we will get some strange behavior.  Although, whatever system we set up should be robust enough to work for future prescribed aerosol models too.  Presumably, SPA won't be the one and only in the future.  I can think of other runtime arrangements where multiple parameters would need to be tweaked and having some dependency might be nice.

**Comment by AaronDonahue** [comment]
Closing, solved by #1715
